orm/db_mysql.go

```diff
@@ -22,7 +22,7 @@ import (
 
 // mysql operators.
 var mysqlOperators = map[string]string{
-	"exact":     "= ?",
+	"exact":     "= BINARY ?",
```

Comment:
I think the better solution is adding a new operator.

Reply:
Hi, I have added a MySQL operator `strictexact` to implement `BINARY` keyword. Please kindly advice on the naming, related testing, etc. Thank you.